uint8 fiducial_id=25

---